### ISSUE TITLE
Different logging with official openpose

### ISSUE THREAD

**Issue by zsdonghao** [tensorlayer/openpose-plus#67]
In the logging of official openpose, the `loss` is the sum of all losses of all branches:

```
I0911 18:24:34.281644 18177 solver.cpp:228] Iteration 599975, loss = 382.516
I0911 18:24:34.281685 18177 solver.cpp:244]     Train net output #0: loss_stage1_L1 = 98.4054 (* 1 = 98.4054 loss)
I0911 18:24:34.281697 18177 solver.cpp:244]     Train net output #1: loss_stage1_L2 = 34.9568 (* 1 = 34.9568 loss)
I0911 18:24:34.281704 18177 solver.cpp:244]     Train net output #2: loss_stage5_L1 = 92.2677 (* 1 = 92.2677 loss)
I0911 18:24:34.281713 18177 solver.cpp:244]     Train net output #3: loss_stage5_L2 = 33.2778 (* 1 = 33.2778 loss)
I0911 18:24:34.281721 18177 solver.cpp:244]     Train net output #4: loss_stage6_L1 = 91.2305 (* 1 = 91.2305 loss)
I0911 18:24:34.281729 18177 solver.cpp:244]     Train net output #5: loss_stage6_L2 = 32.3784 (* 1 = 32.3784 loss)
I0911 18:24:34.281736 18177 sgd_solver.cpp:106] Iteration 599975, lr = 4.91855e-07
I0911 18:24:35.087828 18177 solver.cpp:228] Iteration 599980, loss = 231.808
I0911 18:24:35.087862 18177 solver.cpp:244]     Train net output #0: loss_stage1_L1 = 60.8041 (* 1 = 60.8041 loss)
I0911 18:24:35.087868 18177 solver.cpp:244]     Train net output #1: loss_stage1_L2 = 21.7501 (* 1 = 21.7501 loss)
I0911 18:24:35.087873 18177 solver.cpp:244]     Train net output #2: loss_stage5_L1 = 55.8138 (* 1 = 55.8138 loss)
I0911 18:24:35.087877 18177 solver.cpp:244]     Train net output #3: loss_stage5_L2 = 19.9663 (* 1 = 19.9663 loss)
I0911 18:24:35.087884 18177 solver.cpp:244]     Train net output #4: loss_stage6_L1 = 54.2236 (* 1 = 54.2236 loss)
I0911 18:24:35.087889 18177 solver.cpp:244]     Train net output #5: loss_stage6_L2 = 19.2508 (* 1 = 19.2508 loss)
I0911 18:24:35.087894 18177 sgd_solver.cpp:106] Iteration 599980, lr = 4.91855e-07
I0911 18:24:35.923229 18177 solver.cpp:228] Iteration 599985, loss = 335.47
I0911 18:24:35.923282 18177 solver.cpp:244]     Train net output #0: loss_stage1_L1 = 88.7421 (* 1 = 88.7421 loss)
I0911 18:24:35.923290 18177 solver.cpp:244]     Train net output #1: loss_stage1_L2 = 27.7794 (* 1 = 27.7794 loss)
I0911 18:24:35.923295 18177 solver.cpp:244]     Train net output #2: loss_stage5_L1 = 85.2865 (* 1 = 85.2865 loss)
I0911 18:24:35.923301 18177 solver.cpp:244]     Train net output #3: loss_stage5_L2 = 25.1282 (* 1 = 25.1282 loss)
I0911 18:24:35.923306 18177 solver.cpp:244]     Train net output #4: loss_stage6_L1 = 83.6175 (* 1 = 83.6175 loss)
I0911 18:24:35.923311 18177 solver.cpp:244]     Train net output #5: loss_stage6_L2 = 24.917 (* 1 = 24.917 loss)
I0911 18:24:35.923316 18177 sgd_solver.cpp:106] Iteration 599985, lr = 4.91855e-07
I0911 18:24:36.637693 18177 solver.cpp:228] Iteration 599990, loss = 296.236
I0911 18:24:36.637727 18177 solver.cpp:244]     Train net output #0: loss_stage1_L1 = 70.544 (* 1 = 70.544 loss)
I0911 18:24:36.637734 18177 solver.cpp:244]     Train net output #1: loss_stage1_L2 = 32.5959 (* 1 = 32.5959 loss)
I0911 18:24:36.637739 18177 solver.cpp:244]     Train net output #2: loss_stage5_L1 = 66.6619 (* 1 = 66.6619 loss)
I0911 18:24:36.637743 18177 solver.cpp:244]     Train net output #3: loss_stage5_L2 = 31.0486 (* 1 = 31.0486 loss)
I0911 18:24:36.637749 18177 solver.cpp:244]     Train net output #4: loss_stage6_L1 = 65.2246 (* 1 = 65.2246 loss)
I0911 18:24:36.637753 18177 solver.cpp:244]     Train net output #5: loss_stage6_L2 = 30.1618 (* 1 = 30.1618 loss)
I0911 18:24:36.637758 18177 sgd_solver.cpp:106] Iteration 599990, lr = 4.91855e-07
I0911 18:24:37.321897 18177 solver.cpp:228] Iteration 599995, loss = 343.375
I0911 18:24:37.321946 18177 solver.cpp:244]     Train net output #0: loss_stage1_L1 = 89.8674 (* 1 = 89.8674 loss)
I0911 18:24:37.321954 18177 solver.cpp:244]     Train net output #1: loss_stage1_L2 = 29.9036 (* 1 = 29.9036 loss)
I0911 18:24:37.321992 18177 solver.cpp:244]     Train net output #2: loss_stage5_L1 = 85.3504 (* 1 = 85.3504 loss)
I0911 18:24:37.322000 18177 solver.cpp:244]     Train net output #3: loss_stage5_L2 = 27.5036 (* 1 = 27.5036 loss)
I0911 18:24:37.322008 18177 solver.cpp:244]     Train net output #4: loss_stage6_L1 = 83.8315 (* 1 = 83.8315 loss)
I0911 18:24:37.322016 18177 solver.cpp:244]     Train net output #5: loss_stage6_L2 = 26.9185 (* 1 = 26.9185 loss)
I0911 18:24:37.322024 18177 sgd_solver.cpp:106] Iteration 599995, lr = 4.91855e-07
```

But in our version, the loss is not the sum of all stage losses, which one is correct?

```
Total Loss at iteration 348546 / 600000 is: 326.2651062011719 Learning rate 4.435560e-06 weight_norm 2.053057e+00 Took: 0.7935836315155029s
Network# 0 For Branch 1 Loss: 80.05341
Network# 1 For Branch 2 Loss: 136.94666
Network# 2 For Branch 1 Loss: 79.45871
Network# 3 For Branch 2 Loss: 136.63705
Network# 4 For Branch 1 Loss: 79.07562
Network# 5 For Branch 2 Loss: 136.25262
Total Loss at iteration 348547 / 600000 is: 222.1693572998047 Learning rate 4.435560e-06 weight_norm 2.053057e+00 Took: 0.35128092765808105s
Network# 0 For Branch 1 Loss: 57.39211
Network# 1 For Branch 2 Loss: 92.5421
Network# 2 For Branch 1 Loss: 53.668625
Network# 3 For Branch 2 Loss: 91.577156
Network# 4 For Branch 1 Loss: 54.71547
Network# 5 For Branch 2 Loss: 90.337135
Total Loss at iteration 348548 / 600000 is: 228.1260223388672 Learning rate 4.435560e-06 weight_norm 2.053057e+00 Took: 0.743701696395874s
Network# 0 For Branch 1 Loss: 58.997124
Network# 1 For Branch 2 Loss: 93.80566
Network# 2 For Branch 1 Loss: 57.302673
Network# 3 For Branch 2 Loss: 93.5231
Network# 4 For Branch 1 Loss: 56.936867
Network# 5 For Branch 2 Loss: 91.580536
```
